core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -816,6 +833,10 @@ class SparkContext(config: SparkConf) extends Logging {
   /**
    * Read a text file from HDFS, a local file system (available on all nodes), or any
    * Hadoop-supported file system URI, and return it as an RDD of Strings.
+   *
+   * @param path URI to Hadoop-supported file system
```

Comment:
Well, it's a URI representing a path on a supported file system. But the latter fact was already in the docs. The meaningful doc here is that it's the path of the text file!